app/queries/boards/get_last_boards_in_history_by_role.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+module Boards
+  class GetLastBoardsInHistoryByRole
+    attr_reader :role, :user_id
+
+    def initialize(role, user_id)
+      @role = role
+      @user_id = user_id
+    end
+
+    def call
+      Board.where.not(id: Board.select(:previous_board_id).where.not(previous_board_id: nil))
```

Comment:
We are moving away from using membership in favour of permissions: https://github.com/cybergizer-hq/retrospective/pull/225

Looks like we should find Boards for the user through board permissions with identifier update_board (or something else from CREATOR_IDENTIFIERS)